### PR TITLE
Bing translate

### PR DESCRIPTION
**Pull request by ankit-m**
### Bing Translate

This PR adds support for using **Microsoft Translator (Bing)**. It includes the following features

1. Addition to `bingTranslate.js` in `services`
2. Updating UI for entering two keys based on context
3. Update Note to inform users about translators
4. Add tooltips
5. Create `languages.js` to store all language codes for different translators.

### CODE REVIEW

File: lib/scripts/utils/languages.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+export var bingLanguages = {
```
Comment (Ceilican) [inline lib/scripts/utils/languages.js]: Is it possible to fetch the list of languages supported by Yandex and Bing dynamically? Would it make sense to do it? Or not?

**Comment by Ceilican** [comment]
Thanks!

File: lib/scripts/utils/languages.js (hunk 1 above)
Comment (ankit-m) [inline lib/scripts/utils/languages.js]: I did think of that. But supported languages don't change that often. So we can manually set them.

Comment (Ceilican) [inline lib/scripts/utils/languages.js]: Ok. That's a good point.

**Pull request merged**